versioned_docs/version-8.5/self-managed/modeler/web-modeler/configuration/ssl.md

```diff
@@ -0,0 +1,102 @@
+---
+id: ssl
+title: "SSL"
+description: "Read details on additional SSL configuration for Web Modeler."
+---
+
+By default, communication between Web Modeler and Identity and the Web Modeler components is not encrypted, as it usually happens backend-to-backend within the same [Docker](/self-managed/setup/deploy/other/docker.md) network or [Kubernetes](/self-managed/setup/install.md) cluster.
+However, you can enable TLS-encrypted communication by following the steps below (for example, if backend-to-backend communication is not possible in a custom Camunda 8 installation setup).
+
+## Configuring secure connections to Identity
+
+### Configure the Identity base URL
+
+Provide a URL that starts with `https://` (e.g. `https://identity.example.com`) as the base URL of the Identity instance:
```

Comment:
Very small styling note in that we don't use latin abbreviations anymore @conceptualshark 😄 

<img width="988" alt="Screenshot 2024-06-12 at 9 03 47 AM" src="https://github.com/camunda/camunda-docs/assets/84338309/3b3153eb-8b96-46a1-922d-6a86808ca211">